src/api/index.ts

```diff
@@ -1,5 +1,22 @@
-import { TokenApi, Network } from 'types'
+import { Network, TokenApi, DepositApi } from 'types'
 import { TokenApiImpl } from './TokenApi/TokenApiImpl'
+import { DepositApiMock } from './ExchangeApi/mock/DepositApiMock'
+import balanceStates from '../../test/data/balanceStates'
+
+const isMock = process.env.MOCK === 'true'
+
+function createTokenApi(): TokenApi {
+  return new TokenApiImpl([Network.Mainnet, Network.Rinkeby])
+}
+
+function createDepositApi(): DepositApi {
+  if (true || isMock) {
+    return new DepositApiMock({ balanceStates })
+  } else {
+    throw new Error('Only has been implemented')
```

Comment:
In this case, I think you could either have the `true` in the if condition or throw the error. Because the `else` will never be reached, thus the error is not necessary.

Also, the error message is probably missing a word.

Reply:
Yep, this `true` was a test that I forgot from to remove...

I realised only when I did this other PR: https://github.com/gnosis/dex-react/pull/57
Since they go in serie (every PR uses the previous one as it's base),I fixed it in #57